centreon/src/Core/Dashboard/Application/UseCase/FindPerformanceMetricsData/PerformanceMetricsDataFactory.php

```diff
@@ -253,8 +253,8 @@ private function createMetricInformations(array $metricData, array $metricNames)
                     $metric['warn_low'] !== null ? (float) $metric['warn_low'] : null,
                     $metric['crit'] !== null ? (float) $metric['crit'] : null,
                     $metric['crit_low'] !== null ? (float) $metric['crit_low'] : null,
-                    $metric['ds_color_area_warn'],
-                    $metric['ds_color_area_crit']
+                    $metric['ds_color_area_warn'] ?? $dsData['ds_color_area_warn'],
+                    $metric['ds_color_area_crit'] ?? $dsData['ds_color_area_crit']
```

Comment:
```suggestion
                    $metric['ds_color_area_warn'] ?? $dsData['ds_color_area_warn'] ?? '',
                    $metric['ds_color_area_crit'] ?? $dsData['ds_color_area_crit'] ?? ''
```

Reply:
There is always one of the two that is defined.

Reply:
Done